### PR TITLE
Adjusts shrapnel to cause HALLOSS

### PR DESCRIPTION
**Pull request by skull132**
Shrapnel no longer causes damage. This is to make this shit less lethal, and to increase the survivability of combat when ballistics are in play. They do cause HALLOSS now, though. Which should still result in the person getting filled with shrapnel being immobilized and effectively useless in combat.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -1136,10 +1136,7 @@
 						"<span class='warning'>Your movement jostles [O] in your [organ.name] painfully.</span>")
 					to_chat(src, msg)
 
-				organ.take_damage(rand(1,3), 0, 0)
-				if(!(organ.status & ORGAN_ROBOT) && !(species.flags & NO_BLOOD)) //There is no blood in protheses.
-					organ.status |= ORGAN_BLEEDING
-					src.adjustToxLoss(rand(1,3))
+				src.adjustHalLoss(rand(1, 3))
```
Comment (skull132) [inline code/modules/mob/living/carbon/human/human.dm]:
```suggestion
				adjustHalLoss(rand(1, 3))
```

**Comment by NonQueueingMatt** [comment]
Pain damage heals at a decent speed. Consider that every piece of shrapnel will add halloss every time it procs, and that it also procs pretty often.

**Comment by skull132** [comment]
> Doesn't pain damage heal a lot faster than brute damage? Don't you need to make the pain value higher for it to be relevant?

It kinda depends. First, this stacks. Second, this is ran every time you move. You're unlikely to end up with 1 shrap in you. Say you have 3, which on average deal like 5 total damage per move. You're crawling along after about 5 moves at that point.

**Comment by Arrow768** [comment]
It might need additional adaptations in relation to IPCs given that they can now basically ignore shrapnel.

**Pull request merged**